### PR TITLE
Fix Broken Images In Landing Page

### PR DESCRIPTION
**Pull request by searleser97**
# Context

For simplicity, to show images we were using dynamic links from the internet like Facebook and LinkedIn profile pictures.
Eventually, these links broke, hence our site was not showing images.

# What does this PR do?

Adds images to the repo and update the corresponding links

# Evidence

image

image

### CODE REVIEW

File: src/utils/constants/landing.ts [hunk 1]
```diff
@@ -7,76 +7,76 @@ export const MENU_SECTIONS = [
...
-      "avatar": "https://media-exp1.licdn.com/dms/image/C5603AQHyX3lRB8Ijzg/profile-displayphoto-shrink_800_800/0/1613790705206?e=1626912000&v=beta&t=443BH5A6LT0JcIlpv1QXiemlVsvLRtLCGVhKdmTNKhw",
+      "avatar": "images/Collaborators/Daniela.jpeg",
       "link": "https://www.linkedin.com/in/daniela-vera-garc%C3%ADa-70766a207/"
   },
-  {
-      "name": "Alberto Esquivel",
-      "text": "",
-      "avatar": "https://scontent-dfw5-1.xx.fbcdn.net/v/t1.6435-9/72650239_2495946783785482_1474312971063656448_n.jpg?_nc_cat=103&ccb=1-3&_nc_sid=09cbfe&_nc_eui2=AeG8-MH7zHGb6CEnoaJjAY4poOIoHGAGhNmg4igcYAaE2cqmQwUiufbUdsUR1DQwxhc&_nc_ohc=sHg2kBk3-mUAX-LY6aK&_nc_ht=scontent-dfw5-1.xx&oh=8a470b010ae60f0903661eceeb077427&oe=60C91ACA",
-      "link": "https://www.linkedin.com/in/aesquivelv/"
-  },
```
Comment (akotadi) [inline src/utils/constants/landing.ts:75]: Why did you erase Alberto?

Comment (searleser97) [reply src/utils/constants/landing.ts:75]: I moved him to the top, is not deleted, He doesn't have any profile picture. or at least I couldn't find any and moving it to the top makes him visible only if people scroll all the way to the left.

Comment (akotadi) [reply src/utils/constants/landing.ts:75]: Oh I didn't see that, makes sense!

Comment (searleser97) [reply src/utils/constants/landing.ts:75]: Also, if you could help finding images for the missing people , I'll appreciate it

**Review by akotadi — APPROVED** [review]
LGTM!

**Pull request merged**